apps/web/src/app/auth/settings/security/change-kdf/change-kdf-confirmation.component.ts

```diff
@@ -28,13 +28,12 @@ export class ChangeKdfConfirmationComponent {
   loading = false;
 
   constructor(
-    private apiService: ApiService,
     private i18nService: I18nService,
-    private keyService: KeyService,
     private messagingService: MessagingService,
     @Inject(DIALOG_DATA) params: { kdf: KdfType; kdfConfig: KdfConfig },
     private accountService: AccountService,
     private toastService: ToastService,
+    private changeKdfService: DefaultChangeKdfService,
```

Comment:
```suggestion
    private changeKdfService: ChangeKdfService,
```
Shouldn't we use the abstraction here?